### DISCUSSION TITLE
Provide lovelace yaml config?

### DISCUSSION THREAD

**Discussion by cygnusb** [marq24/ha-evcc#23] [General]
Could you please provide your lovelace yaml, which is shown in the png? This would allow to simply copy your layout, which is arranged perfectly. I am aware that i need to change the entity names. But some small sed action should do the trick.

Thanks for providing this integration!

**Comment by marq24** [comment]
I have to admit, that I don't have the config any longer - since this screenshot was made from my test system - which will be frequently wiped...let me see, if I can reconstruct this...

**Reply by marq24** [reply]
... ok after I have spend now some minutes with the reconstruction - IMHO it's "useless" when I share a config here - since a lot of the entities will include the loadpoint name in the sensor entity name...

When I remember correctly I quicky created the over view by adding every section of the integration overview page twice to the dashbord and then remove on the one copy the loadpoint entities and on the other copy the site related entities...

**Reply by cygnusb** [reply]
I simply can rename the loadpoint in the source to my load point name. So if you have something and can post it, please do. If this is too much work then not.

**Reply by marq24** [reply]
IF I had something already I would have shared it... [I just wanted to prepare it - and "then" I realized, that all the loadpoint entities have my ''hh7' in the entity names...
